src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -1335,6 +1335,21 @@ bool RegisterAppInterfaceRequest::IsApplicationSwitched() {
 
   LOG4CXX_DEBUG(logger_,
                 "Application with policy id " << policy_app_id << " is found.");
+
+  // Checking that isn't other application with same app id and app name
```

Comment:
@ValeriiMalkov Chack English grammar

Reply:
@AByzhynar Fixed in commit e070e1c1375e533c65ef459ded6f56d55cf8651f